src/cow-react/modules/advancedOrders/state/advancedOrdersAtom.ts

```diff
@@ -0,0 +1,43 @@
+import { OrderKind, SupportedChainId } from '@cowprotocol/cow-sdk'
+import { getDefaultTradeState } from '@cow/modules/trade/types/TradeState'
+import { atomWithStorage, createJSONStorage } from 'jotai/utils'
+import { atom } from 'jotai'
+
+export interface AdvancedOrdersState {
```

Comment:
this might change a bit I believe, but I think is fine to have this as a first version

Reply:
It's already changed :) I will open the next PR